### ISSUE TITLE
Most of the translations are not working.

### ISSUE THREAD

**Issue by laurakezbere** [Adyen/adyen-magento2#523]
**Magento version**: 2.3.2
**Plugin version**: 4.5.1
**Description**

Not all translations are working for me. I wrote a mixin for adyen-cc-method.js, rewrote initialize function and replaced this:
```
this.checkout = new AdyenCheckout({
    locale: this.getLocale(),
    risk: {
        enabled: false
    }
});
```
with this:
```
translations[this.getLocale()] = {
    'creditCard.numberField.placeholder': $.mage.__('Card number'),
    'creditCard.holderName.placeholder': $.mage.__('Name on card'),
    'creditCard.cvcField.placeholder': $.mage.__('CVC'),
    'creditCard.expiryDateField.month.placeholder': $.mage.__('Month'),
    'creditCard.expiryDateField.year.placeholder': $.mage.__('Year')
};

// initialize adyen component for general use
this.checkout = new AdyenCheckout({
    locale: this.getLocale(),
    risk: {
        enabled: false
    },
    translations: translations
});
```
To do this I was referencing this documentation - https://docs.adyen.com/checkout/web-sdk/customization/localization Checkout fields section.
So the issue is that this worked only for `creditCard.holderName.placeholder` field and I'm not sure if this is an issue on my side or the documentation has provided wrong keys.

**Comment by cyattilakiss** [comment]
Hi @laurakezbere ,

Thank you for raising this issue. Indeed, we are currently using the checkout component version 3.0.0 and this issue that you mentioned was fixed in the v.3.2.0.
We are going to implement that version asap.

Best,
Attila
Adyen

**Comment by cyattilakiss** [comment]
Hi @laurakezbere ,

In the linked PR above we bumped the version of the checkout component to 3.2.0. You can use this branch or wait for the next release when this will be released as well.

Best,
Attila
Adyen

**Comment by cyattilakiss** [comment]
Hi @laurakezbere ,

We have released version 5.0.0 including the fix for your issue.
Let me know if you face any issue regarding to this.

Best,
Attila
Adyen
